sw/ground_segment/python/gvf/gvfFormation.py

```diff
@@ -7,9 +7,9 @@
 import numpy as np
 import sys
 from os import path, getenv
-PPRZ_SRC = getenv("PAPARAZZI_SRC", path.normpath(path.join(path.dirname(path.abspath(__file__)), '../../../../')))
-sys.path.append(PPRZ_SRC + "/sw/lib/python")
-sys.path.append(PPRZ_SRC + "/sw/ext/pprzlink/lib/v1.0/python")
+PPRZ_HOME = getenv("PAPARAZZI_HOME", path.normpath(path.join(path.dirname(path.abspath(__file__)), '../../../../')))
+sys.path.append(PPRZ_HOME + "/sw/lib/python")
+sys.path.append(PPRZ_HOME + "/var/lib/python")
```

Comment:
Actually, only this one should use PPRZ_HOME.
Things in conf/ and var/ should use PAPARAZZI_HOME, everything else (sw, doc, data, ...) should use PAPARAZZI_SRC. That is for the day we fix the binary debian packaging for paparazzi...